QUARKUS-508.md

```diff
@@ -0,0 +1,46 @@
+# QUARKUS-508 Create one or more quarkus devfile v2, ensure odo based workflow
+
+Jira: https://issues.redhat.com/browse/QUARKUS-508
+
+The goal of QUARKUS-508 is to provide a runtime descriptor for OpenShift development tools (e.g. odo, Eclipse Che).
+For purposes of this feature request, odo based workflow will be considered.
+
+## Scope of the testing
+
+- Define the scope of the testing
+- Analyze the possible impact on Quarkus itself - e.g performance, compatibility
+- Determine impact on test suites - e.g. new test suite, increased complexity of existing test suite, maintenance burden
+- Determine impact on QE infrastructure - e.g. increased complexity of existing automation, maintenance burden, need for new external resources
+
+### Impact on testsuites and testing automation:
+A test for starter application created and deployed by the `java-quarkus` component devfile will be implemented. This 
+test will be ran in a separate, new job as a part of post-GA tasks, verifying that the `redhat-product` starter 
+application defined in the `devfile` produces the correct productised starter application.
+
+A deployment strategy utilising `odo` tool will be implemented in [`quarkus-test-framework`](https://github.com/quarkus-qe/quarkus-test-framework/).
+
+A test using the `odo` deployment strategy will deploy a simple REST endpoint and verify that the endpoint responds.
+This test will be implemented in [`quarkus-test-suite`](https://github.com/quarkus-qe/quarkus-test-suite/) project.
+
+### Impact on resources:
+Jenkins workers will need to be updated with `odo` tool.
```

Comment:
OC tool is downloaded as part of the job execution. Why do we want to have `odo` as standard part of worker node. It can get old over time if the image is not regularly updated.

Reply:
Well, there's two options:

- download as part of job execution
- install by yum

I'm not really particular to either